api/src/main/java/net/md_5/bungee/api/plugin/PluginManager.java

```diff
@@ -429,4 +435,27 @@ public void unregisterListeners(Plugin plugin)
             it.remove();
         }
     }
+
+    // Checks for plugin updates from plugin folder and performs update
+    private void checkForPluginUpdate(File pluginsFolder, File pluginFile)
+    {
+        File updateFolder = new File( pluginsFolder, "update" );
```

Comment:
This is actually different to Bukkit, this puts the update folder in plugins/update, whereas I'm pretty sure Bukkit is just update

Reply:
No, see [Bukkit's SimplePluginManager](https://hub.spigotmc.org/stash/projects/SPIGOT/repos/bukkit/browse/src/main/java/org/bukkit/plugin/SimplePluginManager.java#103-112)

Reply:
Well that goes to show the popularity of the Bukkit feature